src/main.rs

```diff
@@ -176,6 +174,8 @@ fn protected_main() {
     start_bridge();
     start_editor();
     create_window();
+
+    maybe_disown();
```

Comment:
I think this change delays the forking behavior until after the main program has been run, as `create_window` takes control over the thread using `EventLoop::run`, so essentially disables forking. Since the intention was to detect startup issues in console without having to run again with `--nofork`, moving it before control is taken over might help:
```suggestion
    start_bridge();
    start_editor();
    maybe_disown();
    
    create_window();
```

Reply:
The fix I suggested doesn't display the message anymore, since the Neovim runtime is spawned on a separate thread to be handled by tokio. I guess I might move creation of the command before the thread is spawned, and commit that one, too (after calming rustfmt).

Reply:
Fixed that.